doc/source/train/deepspeed.rst

```diff
@@ -0,0 +1,33 @@
+.. _train-deepspeed:
+
+Training with DeepSpeed
+=======================
+
+Ray :class:`~ray.train.torch.TorchTrainer` can help you easily distribute your DeepSpeed training over a Ray Cluster.
+Simply put all your existing training logics into a training function, then launch a TorchTrainer.
+
+To run DeepSpeed with pure PyTorch, you **don't need to** provide any additional Ray Train utilities 
+like :meth:`~ray.train.torch.prepare_model` or :meth:`~ray.train.torch.prepare_data_loader` in your training funciton. Instead, 
```

Comment:
We can move this part as a note/tip after the code block, as this is secondary. The key point to make is that you use a TorchTrainer + DeepSpeed.

Reply:
Yeah it's a supplementary notes. I've put it after the code example